pkg/verrazzano/l10n/en-us.yaml

```diff
@@ -1458,6 +1458,7 @@ verrazzano:
     thanosQueryUrl: Thanos Query
     slack: Slack
     title: Verrazzano Links
+    alertmanagerUrl: Alertmanager Console
```

Comment:
We went to the trouble of alphabetizing these keys (within their section) we added to make them easier to see/find.  Please do the same.

Reply:
done